docs/UserGuide.md

```diff
@@ -97,18 +97,20 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [sch/SCHOOL] [stream/ACAD_STREAM] [lvl/ACAD_LEVEL] [r/REMARK] [t/TAG]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
+* You can delete the data for optional fields by typing sch//stream//lvl/ without specifying content after the prefixes.  
```

Comment:
@Xiaoyunnn "prefix" is a technical term. Maybe you can rephrase it or add into glossary

Reply:
oh wait UG has no glossary.. ok you can ignore this first, maybe I'll add something into command format notes later